bindings/rust/bench/historical-perf/bench-past.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# suppress stdout
+exec >/dev/null
+export CARGO_TERM_QUIET=true
+export RUSTFLAGS=-Awarnings
+
+# immediately bail if any command fails
+set -e
+
+# go to s2n-tls/bindings/rust/bench/
+bench_path=`pwd`/`dirname "$0"`/../
+pushd $bench_path
+
+# make Cargo.toml point s2n-tls to the cloned old version
+sed -i "s|s2n-tls = .*|s2n-tls = { path = \"target/s2n-tls/bindings/rust/s2n-tls\" }|" Cargo.toml 
+
+# ensure Cargo.toml gets changed back on exit; retains original exit status
+trap "{ status=$?; sed -i 's|s2n-tls = .*|s2n-tls = { path = \"../s2n-tls\" }|' $bench_path/Cargo.toml; exit $status; }" EXIT
+
+# clone copy of repo to target/s2n-tls
+echo "cloning repo" >&2
+mkdir -p target
+cd target
+rm -rf s2n-tls
+git clone --quiet https://github.com/aws/s2n-tls
+cd s2n-tls/bindings/rust/
+copied_bindings_path=`pwd`
+
+# last tag we want is v1.3.16, get all tags from then
+# `git tag -l | sort -rV` gets list of sorted tags newest to oldest
```

Comment:
dead code?

Reply:
This is just to explain how the bash things work a little more clearly and break down the long piped commands. I wouldn't say it's dead code but just part of a comment.